src/Sarif/Baseline/ResultMatching/SarifLogResultMatcher.cs

```diff
@@ -278,6 +278,7 @@ private List<ExtractedResult> ExtractResultsFromRuns(IEnumerable<Run> sarifRuns)
             run.Graphs = graphs;
             run.Invocations = invocations;
             run.Properties = properties;
+            run.VersionControlProvenance = MergeVersionControlDetails(currentRuns.Concat(previousRuns));
```

Comment:
>MergeVersionControlDetails [](http://example.com/codeflow?start=43&length=26)

can you verify if we have other relevant properties that we should keep?

Reply:
there are couple properties of run like Addresses/WebRequest/WebResponse etc are not been set.
I am not sure if any special requirements to merge these properties. Can I just combine the values of the properties in the final run object?

Reply:
moved this change to another PR so each PR. resolving this comment.